specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StorageTask"
+            },
+            "description": "The parameters to create a Storage Task."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Update storage task with the same properties specified in the request body.",
+            "schema": {
+              "$ref": "#/definitions/StorageTask"
+            }
+          },
+          "201": {
+            "description": "OK -- Create storage task with the same properties specified in the request body.",
+            "schema": {
+              "$ref": "#/definitions/StorageTask"
+            }
+          },
+          "202": {
+            "description": "Accepted -- Create or update request accepted; operation will complete asynchronously.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "delete": {
+        "tags": [
+          "StorageTasks"
+        ],
+        "description": "Delete the storage task resource.",
+        "operationId": "StorageTasks_Delete",
+        "x-ms-examples": {
+          "PutStorageTask": {
+            "$ref": "./examples/storageTasksCrud/DeleteStorageTask.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageTaskName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted -- Delete request accepted; operation will complete asynchronously.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
```

Comment:
>204 [](http://example.com/codeflow?start=11&length=3)

Also add 200 response for resource deleted successfully, as 204 is for resource didn't exist.

Reply:
On adding 200 ; Below is the failure. I will be reverting the change
https://github.com/Azure/azure-openapi-validator/blob/main/docs/delete-response-codes.md#output-message

Reply:
for LRO delete operations it expects 202 and 204 only and no 200. Let me know if I should remove 200

Reply:
As per discussion, removing 200 status code